src/Controller/Component/PaginatorComponent.php

```diff
@@ -282,7 +282,7 @@ protected function _setPagingParams()
      */
     public function config($key = null, $value = null, $merge = true)
     {
-        $return = $this->_paginator->config($key, $value, $merge);
+        $return = $this->_paginator->setConfig($key, $value, $merge);
```

Comment:
This change is incorrect, as `config()` is really used as a combined setter/getter here. See the following code in the method. If we want to remove this call, probably the fix will be like this:
```php
public function config($key = null, $value = null, $merge = true)
{
    if (is_array($key) || func_num_args() >= 2) {
        $this->_paginator->setConfig($key, $value, $merge);

        return $this;
    }

    return $this->_paginator->getConfig($key);
}
```
Though I touched this method a little bit in my PR (#10959), feel free to fix it. I will rebase my commit then.